src/Compilers/CSharp/Portable/Symbols/Source/ThisParameterSymbol.cs

```diff
@@ -173,5 +173,7 @@ internal override bool IsMetadataOut
         internal override ImmutableArray<int> InterpolatedStringHandlerArgumentIndexes => ImmutableArray<int>.Empty;
 
         internal override bool HasInterpolatedStringHandlerArgumentError => false;
+
+        internal override DeclarationScope Scope => DeclarationScope.Unscoped;
```

Comment:
>DeclarationScope.Unscoped [](http://example.com/codeflow?start=52&length=25)

Is this the right value for `this` parameter? The spec says the following: "scoped ref T is the default type of this."